cmd/wallet-web/src/components/CredentialPreview/CredentialPreview.vue

```diff
@@ -42,5 +69,25 @@ export default {
       required: true,
     },
   },
+  computed: {
+    gradientColor() {
+      switch (this.category) {
+        case 'pr-card':
+          return 'boatBlue';
```

Comment:
Would it be possible to define gradient color in the config file itself rather than adding it vue file here ?

Reply:
This way we would have everything related to credential display in one file (for now, config json file in codebase)

Reply:
I would agree, as we discussed please move this brandcolor to config. otherwise looks nice

Reply:
Addressed @rolsonquadras @talwinder50